specification/network/resource-manager/Microsoft.Network/2017-06-01/expressRouteConnector.json

```diff
@@ -0,0 +1,372 @@
+		"description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+		"version": "2017-06-01",
+		"title": "NetworkManagementClient"
+	},
+	"host": "management.azure.com",
+	"schemes":[
+		"https"
+	],
+	"consumes": [
+		"application/json"
+	],
+	"produces": [
+		"application/json"
+	],
+	"security":[
+		{
+			"azure_auth": [
+				"user_impersonation"
+			]
+		}
+	],
+	"securityDefinitions": {
+		"azure_auth": {
+			"type":"oauth2",
+			"authorizationUrl":"https://login.microsoftonline.com/common/oauth2/authorize",
+			"flow":"implicit",
+			"description":"Azure Active Directory OAuth2 Flow",
+			"scopes": {
+				"user_impersonation":"impersonate your user account"
+			}
+		}
+	},
+	"paths":{
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/expressRouteConnectors/{expressRouteConnectorName}": {
+			"delete": {
+				"tags": [
+					"expressRouteConnectors"
+				],
+				"operationId": "expressRouteConnector_Delete",
+				"description": "Deletes the specified Express Route Connector Resource.",
+				"x-ms-examples": {
+                    "ExpressRouteConnectorDelete": { "$ref": "./examples/ExpressRouteConnectorDelete.json" }
+                },
+				"parameters": [
+					{
+						"$ref": "#/parameters/ResourceGroupNameParameter"
+					},
+					{
+						"$ref": "#/parameters/ExpressRouteConnectorNameParameter"
+					},
+					{
+						"$ref": "#/parameters/ApiVersionParameter"
+					},
+					{
+						"$ref": "#/parameters/SubscriptionIdParameter"
+					}
+				],
+				"responses": {
+					"202": {
+						"description": "Accepted and the operation will complete asynchronously."
+					},
+					"200": {
+						"description": "Delete successful."
+					},
+					"204": {
+						"description": "Delete successful."
+					}
+				},
+				"x-ms-long-running-operation": true
+			},
+			"get": {
+				"tags": [
+					"expressRouteConnector"
+				],
+				"operationId": "expressRouteConnector_Get",
+				"x-ms-examples": {
+                    "RouteFilterGet": { "$ref": "./examples/ExpressRouteConnectorGet.json" }
+                },
+				"description": "Gets the specfic Express Route Connector Resource from the specified resource group.",
+				"parameters": [
+					{
+						"$ref": "#/parameters/ResourceGroupNameParameter"
+					},
+					{
+						"$ref": "#/parameters/ExpressRouteConnectorNameParameter"
+					},
+					{
+						"$ref": "#/parameters/ApiVersionParameter"
+					},
+					{
+						"$ref": "#/parameters/SubscriptionIdParameter"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "Request successful. The operation returns the Express Route Connector Resource resource.",
+						"schema": {
+							"$ref": "#/definitions/expressRouteConnector"
+						}
```

Comment:
SDK Team requested me to remove 4xx Error codes.
@fearthecowboy / @olydis / @dsgouda  : Can you comment if we need to add back error codes?